### PR TITLE
Removed obvious and meaningless comments

### PR DESCRIPTION
**Pull request by IAmNotHanni**
I removed all the pointless comments in the code.

### CODE REVIEW

**Comment by Croydon** [comment]
Please don't make a commit for each file. Squash them to one commit.

File: inexor/engine/main.cpp [hunk 1]
```diff
@@ -252,19 +211,12 @@ static void getbackgroundres(int &w, int &h)
...
-/// @see renderbackground
 void restorebackground()
 {
     if(renderedframe) return;
     renderbackground(backgroundcaption[0] ? backgroundcaption : NULL, backgroundmapshot, backgroundmapname[0] ? backgroundmapname : NULL, backgroundmapinfo, true);
 }
 
-
-
-/// Render a textured quad of the given dimensions.
```
Comment (a-teammate) [inline inexor/engine/main.cpp]: We shouldnt simply remove all doc comments but the ones which do not give any extra information.

File: inexor/engine/octaedit.cpp [hunk 2]
```diff
@@ -405,14 +342,9 @@ extern bool hoveringonent(int ent, int orient);
 extern void renderentselection(const vec &o, const vec &ray, bool entmoving);
 extern float rayent(const vec &o, const vec &ray, float radius, int mode, int size, int &orient, int &ent);
 
-/// automaticly select the grid size of the selected face
 VAR(gridlookup, 0, 0, 1);
-
-/// ignore the first cube when view ray intersects it and select the second ("select through cubes")
 VAR(passthroughcube, 0, 1, 1);
 
-/// render selection box to the cursor target
```
Comment (a-teammate) [inline inexor/engine/octaedit.cpp]: selection grid maybe but otherwise valid

File: inexor/fpsgame/aiman.hpp [hunk 3]
```diff
@@ -239,14 +223,11 @@ namespace aiman
 		loopvrev(bots) if(bots[i]) reinitai(bots[i]);
 	}
 
-	// clear and remove all ai immediately
 	void clearai()
 	{ 
         loopvrev(bots) if(bots[i]) deleteai(bots[i]);
 	}
 
-	// check if we do even need AI in this game
```
Comment (a-teammate) [inline inexor/fpsgame/aiman.hpp]: valid, but needs fixed spaces

File: inexor/fpsgame/aiman.hpp [hunk 4]
```diff
@@ -259,21 +240,18 @@ namespace aiman
 		else clearai();
 	}
 
-	// master requires to add a bot
```
Comment (a-teammate) [inline inexor/fpsgame/aiman.hpp]: valid but /// instead of // and player instead of master and spaces fixed

File: inexor/fpsgame/aiman.hpp [hunk 5]
```diff
@@ -259,21 +240,18 @@ namespace aiman
...
 	void reqadd(clientinfo *ci, int skill)
 	{
         if(!ci->local && !ci->privilege) return;
         if(!addai(skill, !ci->local && ci->privilege < PRIV_ADMIN ? botlimit : -1)) sendf(ci->clientnum, 1, "ris", N_SERVMSG, "failed to create or assign bot");
 	}
 
-	// master requires to delete a bot
```
Comment (a-teammate) [inline inexor/fpsgame/aiman.hpp]: see above

File: inexor/fpsgame/aiman.hpp [hunk 6]
```diff
@@ -293,24 +270,18 @@ namespace aiman
         sendservmsg(msg);
     }
 
-	// notify bots that map has been changed
```
Comment (a-teammate) [inline inexor/fpsgame/aiman.hpp]: valid but /// and spaces

File: inexor/fpsgame/aiman.hpp [hunk 7]
```diff
@@ -293,24 +270,18 @@ namespace aiman
...
-	// force server bot manager to refresh bot balance
     void changemap()
     {
         dorefresh = true;
         loopv(clients) if(clients[i]->local || clients[i]->privilege) return;
         if(botbalance != (serverbotbalance != 0)) setbotbalance(NULL, serverbotbalance != 0);
     }
 
-	// a new human player connected
-	// change/refresh the way computer controlled players think
```
Comment (a-teammate) [inline inexor/fpsgame/aiman.hpp]: is the second line correct? or is dorefresh a signal to the botmanager to e.g. remove bots?

maybe shrink that to one catchy line?

File: inexor/fpsgame/aiman.hpp [hunk 8]
```diff
@@ -293,24 +270,18 @@ namespace aiman
...
     void addclient(clientinfo *ci)
     {
         if(ci->state.aitype == AI_NONE) dorefresh = true;
     }
 
-	// a human player has changed team
-	// change/refresh the way computer controlled players think
```
Comment (a-teammate) [inline inexor/fpsgame/aiman.hpp]: same

File: inexor/fpsgame/client.cpp [hunk 9]
```diff
@@ -75,7 +61,6 @@ namespace game
         gle::end();
     }
 
-	/// draw a specific teamate's icon arrow in minimap
```
Comment (a-teammate) [inline inexor/fpsgame/client.cpp]: teammate and fixed spaces but otherwise valid

File: inexor/fpsgame/client.cpp [hunk 10]
```diff
@@ -94,13 +79,11 @@ namespace game
         gle::attribf(bx - bs*v.y, by + bs*v.x); gle::attribf(0, 1);
     }
 
-    /// set specific textures for teammates, skulls... on the minimap
```
Comment (a-teammate) [inline inexor/fpsgame/client.cpp]: valid

File: inexor/fpsgame/client.cpp [hunk 11]
```diff
@@ -333,25 +282,18 @@ namespace game
...
     }
 
-	/// write my player name to a stream
-    /// @see escapestring
     void writeclientinfo(stream *f)
     {
         f->printf("name %s\n", escapestring(player1->name));
     }
 
-    /// check if editing is available
```
Comment (a-teammate) [inline inexor/fpsgame/client.cpp]: valid

**Comment by a-teammate** [comment]
We'll solve this as a side-effect when doing the refactorings

**Pull request closed without merging**